profiles/powersave/tuned.conf

```diff
@@ -9,6 +9,9 @@ summary=Optimize for low power consumption
 governor=ondemand|powersave
 energy_perf_bias=powersave|power
 
+[acpi]
+platform_profile=low-power
```

Comment:
On some devices (including my own, ASUS GA402RJ) the `low-power` option doesn't exist, and is instead `quiet`. I'm not sure how tuned configs handle the case where an option doesn't exist, but maybe putting `low-power|quiet` would solve it? Someone more familiar with tuned than I could probably answer that.

Reply:
Thanks for noticing this. Yes, your proposed solution should work, I updated the profile.